server/utils/applications/shouldShowContingencyPlanPages.ts

```diff
@@ -32,7 +32,7 @@ export const shouldShowContingencyPlanPartnersPages = (application: Application)
   )
     return true
 
-  const pssEndDate = retrieveOptionalQuestionResponseFromFormArtifact(application, EndDates, 'pssDate')
+  const pssEndDate = retrieveOptionalQuestionResponseFromFormArtifact(application, EndDates, 'pssEndDate')
```

Comment:
Can we keep `pssDate` in here too? Just in case a user is half way through an application and already has `pssDate` completed

Reply:
Good thinking

Reply:
I have pushed a change so we check if pssDate exists too